Makefile

```diff
@@ -615,9 +615,8 @@ $(eval $(call FIP_ADD_IMG,BL33,--nt-fw))
 endif
 
 ifeq (${NEED_BL2U},yes)
-BL2U_PATH	:= $(if ${BL2U},${BL2U},$(call IMG_BIN,2u))
-$(if ${BL2U}, ,$(eval $(call MAKE_BL,2u)))
-$(eval $(call FWU_FIP_ADD_PAYLOAD,${BL2U_PATH},--ap-fwu-cfg))
+$(if ${BL2U}, $(eval $(call FIP_ADD_IMG,BL32,--ap-fwu-cfg)),\
```

Comment:
The parameter to FIP_ADD_IMG: is this BL32, or BL2U?

Reply:
Oops - my bad.
BL2U, of course.
I will fix it.  Thanks!